apps/website/src/app/docs/report/page.tsx

```diff
@@ -0,0 +1,232 @@
+            <div>Warn</div>
+            <div>Alert</div>
+            <div>Unknown</div>
+            <div>Loading</div>
+          </div>
+        </div>
+
+        <div className="divide-y divide-gray-100">
+          <div className="grid grid-cols-6 gap-x-6 py-5">
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportIcon />"}</pre>
+            </div>
+
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.pass)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.warn)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.alert)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" hadLoadingError={true} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" />
+            </div>
+          </div>
+          <div className="grid grid-cols-6 gap-x-6 py-5">
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportIcon onClick />"}</pre>
+            </div>
+
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                data={getReportExampleData(Rating.pass)}
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                data={getReportExampleData(Rating.warn)}
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                data={getReportExampleData(Rating.alert)}
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon
+                hadLoadingError={true}
+                name="notrab.eth"
+                onClick={() => alert("Clicked")}
+              />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" onClick={() => alert("Clicked")} />
+            </div>
+          </div>
+        </div>
+
+        <h1 className="justify-center flex font-bold text-2xl mb-8 pt-12">
+          {"<"}ReportBadge {"/>"} documentation
+        </h1>
+
+        <div className="divide-y divide-gray-100">
+          <div className="grid grid-cols-6 gap-x-6 py-5 text-center font-medium">
+            <div></div>
+            <div>Success</div>
+            <div>Warn</div>
+            <div>Alert</div>
+            <div>Unknown</div>
+            <div>Loading</div>
```

Comment:
Why isn't the icon in the loading states for `ReportBadge` softly animated? The loading state of the icon for `ReportIcon` is. `ReportBadge` should do the same.